myning/tui/inventory.py

```diff
@@ -7,18 +7,23 @@
 
 
 class InventoryWidget(DataTable):
+    hash = None
```

Comment:
Is this not a class attribute at this point?

Reply:
you're correct and this was intentional. We currently only have one instance of this widget right now so it doesn't matter whether this is an instance or class attribute, but if we added more instances of this widget (for example if we add it to the mining screen), then we do want the hash to be the same across both instances because the hash is not unique to each widget instance, but unique to the player's inventory which is a singleton. the argument could be made that making it an instance attribute would still achieve the same result, but semantically if the inventory items have changed, then the hash across instances should be updated.